src/containers/BridgeCard/CardHeader.tsx

```diff
@@ -131,19 +149,102 @@ const Header = styled(Box)(() => {
   };
 });
 
-const StyledAdvanceButton = styled(IconButton)(() => {
+export const SBox = styled(Box)(() => {
+  return {
+    h1: {
+      fontSize: '2rem',
+    },
+  };
+});
+
+const StyledMenu = styled((props: MenuProps) => (
+  <Menu
+    elevation={0}
+    anchorOrigin={{
+      vertical: 'bottom',
+      horizontal: 'right',
+    }}
+    transformOrigin={{
+      vertical: 'top',
+      horizontal: 'right',
+    }}
+    {...props}
+  />
+))(() => {
   const { currentTheme } = useCustomTheme();
   return {
-    '&:has(.advance-activated)': {
-      background: currentTheme.ghost[800],
+    '& .MuiPaper-root': {
+      borderRadius: '1.6rem',
+      border: '1px solid',
+      borderColor: currentTheme.steel[700],
+      background: currentTheme.steel[900],
+      color: currentTheme.steel[100],
+      fontSize: '1.6rem',
+      minWidth: '208px',
+      minHeight: '72px',
+      padding: '1rem',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      '.menu-item': {
+        display: 'flex',
+        width: '100%',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: 'var(--spacing-2xl, 1.25rem)',
```

Comment:
we are not using vars, also i think the gap should be bigger :thinking: 
```suggestion
        gap: '1.25rem',
```

Reply:
eb39c05